metrics-openjdk/src/test/java/com/alibaba/metrics/jvm/JmxRemoteMetricsTest.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * FileName: JmxRemoteMetricsTest
+ * Author:   HuangTaiHong
+ * Date:     2019/6/6 14:43
+ * Description: jmx remote metrics test.
+ * History:
+ * <author>          <time>          <version>          <desc>
+ * 作者姓名           修改时间           版本号              描述
+ */
+package com.alibaba.metrics.jvm;
+
+import com.alibaba.metrics.JmxAttributeGauge;
+import com.alibaba.metrics.Metric;
+import com.alibaba.metrics.MetricName;
+
+import javax.management.MBeanServerConnection;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * 〈jmx remote metrics test.〉
+ *
+ * @author HuangTaiHong
+ * @create 2019/6/6
+ * @since 1.0.0
+ */
+public class JmxRemoteMetricsTest {
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+
+    // @Test
+    public void testCollectionRemoteMetrics() throws IOException {
+        Map<String, Object> environment = new HashMap<String, Object>();
+        environment.put(JMXConnector.CREDENTIALS, new String[]{USERNAME, PASSWORD});
+        JMXServiceURL jmxServiceURL = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://localhost:1099/jmxrmi");
+        MBeanServerConnection mBeanServerConnection = JMXConnectorFactory.connect(jmxServiceURL, environment).getMBeanServerConnection();
+        BufferPoolMetricSet bufferPoolMetricSet = new BufferPoolMetricSet(mBeanServerConnection);
+        for (Map.Entry<MetricName, Metric> entry : bufferPoolMetricSet.getMetrics().entrySet()) {
+            JmxAttributeGauge value = (JmxAttributeGauge) entry.getValue();
+            System.out.println(entry.getKey().getKey() + ":" + value.getValue());
```

Comment:
Please mock the data and using assertion to check the expected value and actual value.